package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@yisuses/whemotion-front",
+  "version": "1.0.0",
```

Comment:
this should never change. Nothing happens if it does, but its better for every package to increase on its own